snewpdag/plugins/EvalMap.py

```diff
@@ -65,14 +66,14 @@ def compare(self, keys, tdelay):
     v = self.cache[kc]
     if max_width > 0.0:
       ref_nbins = v.nbins
-      ref_duration = v.duration
-      ref_start = v.start
+      ref_duration = v.xwidth
```

Comment:
the width (of the bin I guess), is not the same as the duration from min to max? what is this ref_duration?